src/javascript/JContent/ContentRoute/ToolBar/SortSelector/SortSelector.jsx

```diff
@@ -0,0 +1,113 @@
+import React, {useMemo} from 'react';
+import PropTypes from 'prop-types';
+import {shallowEqual, useDispatch, useSelector} from 'react-redux';
+import {useTranslation} from 'react-i18next';
+import {cmSetSort} from '~/JContent/redux/sort.redux';
+import {ArrowDown, ArrowUp, Dropdown, CustomDropdown, Separator, MenuItem} from '@jahia/moonstone';
+import classes from './SortSelector.scss';
+import JContentConstants from '~/JContent/JContent.constants';
+
+const SORT_DATA = [
```

Comment:
we should also insert selector roles for each of the dropdown items using `attributes` data object (see `ContentStatusSelector` for example) and use it for the test selectors instead of using labels.